### PR TITLE
Feat/pokemon list

### PR DESCRIPTION
**Pull request by sheilaazhara**
### Overview
- Create pokemon list
- Create pokemon types in pop up filter
- Fix bg color and modal size

### Preview
image
image

### CODE REVIEW

File: src/components/Filter/index.vue [hunk 1]
```diff
@@ -70,8 +89,22 @@ export default {
   data() {
     return {
       name: '',
+      types: [],
+      filterTypes: [],
     };
   },
+  mounted() {
+    axios
+      .get('https://pokeapi.co/api/v2/type')
```
Comment (Ibwedagama) [inline src/components/Filter/index.vue]: hello, @sheilaazhar it's a good practice to store any credentials (including API Endpoint) on `.env` file. The benefit of using Environment Variables are:

1. We can hide the API Endpoint from the source code
2. We can easily change the API Endpoint from the deployment configuration
3. We create a habit of securely storing credentials

My suggestion is to create a `.env` file and add them on `.gitignore`

```bash
# .env
VUE_APP_API_ENDPOINT="https://pokeapi.co/api/v2"
```

we can access the ENV Variable using `process.env`

```js
axios
      .get(`${process.env.VUE_APP_API_ENDPOINT}/type`)
```

ref: https://cli.vuejs.org/guide/mode-and-env.html#environment-variables

File: src/views/HomeView.vue [hunk 2]
```diff
@@ -55,14 +74,41 @@ export default {
...
     FilterIcon,
     ModalFilter,
+    CardPokemon,
   },
   data() {
     return {
       filterOpen: false,
       searchName: '',
+      pokemons: [],
+      apiUrl: 'https://pokeapi.co/api/v2/pokemon',
```
Comment (Ibwedagama) [inline src/views/HomeView.vue]: same as my previous comment, we can use `process.env.VUE_APP_API_ENDPOINT`

File: src/views/HomeView.vue [hunk 3]
```diff
@@ -47,6 +64,8 @@
 import BaseLayout from '@/components/Layout/index.vue';
 import { SearchIcon, FilterIcon } from '@vue-hero-icons/outline';
 import ModalFilter from '@/components/Filter/index.vue';
+import axios from 'axios';
+import CardPokemon from '@/components/Card Pokemon/index.vue';
```
Comment (Ibwedagama) [inline src/views/HomeView.vue]: hello, @sheilaazhar there is a space between component names, it makes the build process throw an error

I think we can remove the space like this:
```js
import CardPokemon from '@/components/CardPokemon/index.vue';
```

File: src/components/Filter/index.vue (hunk 1 above)
Comment (sheilaazhara) [reply src/components/Filter/index.vue]: Oh I see, thank you @Ibwedagama I have fixed [it](https://github.com/sheilaazhar/fe-onboarding-mini-task/pull/4/commits/6f9ba7d8b41018f20ec0c04488fb81df6ed01529)

File: src/views/HomeView.vue (hunk 3 above)
Comment (sheilaazhara) [reply src/views/HomeView.vue]: Oh ya i forgot about it, i have fixed it

File: src/views/HomeView.vue (hunk 2 above)
Comment (sheilaazhara) [reply src/views/HomeView.vue]: ok done, i have fixed it

**Review by agunghide — APPROVED** [review]

**Review by Ibwedagama — APPROVED** [review]

**Pull request merged**